code/game/gamemodes/traitor/traitor.dm

```diff
@@ -345,6 +345,34 @@
 	if(M && M != traitor_mob)
 		to_chat(traitor_mob, "We have received credible reports that [M.real_name] might be willing to help our cause. If you need assistance, consider contacting them.")
 		traitor_mob.mind.store_memory("<b>Potential Collaborator</b>: [M.real_name]")
+		//let's also inform their contact that they might be called upon, but leave it vague.
+		inform_collab(M)
+
+/datum/game_mode/proc/inform_collab(mob/living/carbon/human/M)
+	if(!M)
+		return
+	if(M.mind in traitors)		//if you are already a traitor, you already know the codewords and your role, so skip this message.
+		return
+	//Mad-libs for their message
+	var/adjective = pick("strange", "mysterious", "sinister", "un-assuming", "unexpected")
+	var/action_words = pick("aid the fight against Nanotrasen", "repay a debt", "partake in some mischief", "help overthrow the system", "stick it to the man")

```

Comment:
Maybe worth taking out "aid the fight against Nanotrasen". Your character probably doesn't want to do this if they're *working for* NT. "shake up the system", "score one for the working class", "strengthen the unions" or similar might be a better fit.